src/command_modules/azure-cli-acs/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+2.3.14
```

Comment:
Since 2.3.13 isn't published yet, I think you can just add the release note bullet point to that section.